src/components/BookList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import BookItems from './BookItems';
+// import AddButton from './AddButton';
+
+function BookList() {
+  const booksData = useSelector((state) => state.book.books);
+  return (
+    <div>
+      {booksData.map((book) => (
+        <BookItems key={book.id} book={book} />
+      ))}
+      {/* <AddButton /> */}
```

Comment:
- [ ] Just as I pointed out above about Following the best practices, you are not meant to leave unused code as a comment on your code base. I'll suggest you remove this as it makes your codebase look a bit a bit unprofessional. :+1: :smile: